config/initializers/decidim.rb

```diff
@@ -4,7 +4,7 @@
 Decidim.configure do |config|
   config.application_name = "Decidim Barcelona"
   config.mailer_sender    = Rails.application.secrets.email
-  config.authorization_handlers = [CensusAuthorizationHandler]
+  config.authorization_handlers = ["CensusAuthorizationHandler"]
```

Comment:
Also, the `require_relative` at the top of the file is no longer needed.

Reply:
Cool! By the way, are you everywhere? xDDD